packages/core/src/index.ts

```diff
@@ -217,6 +218,8 @@ function init(options: InitOptions): OnboardAPI {
     unstoppableResolution
   })
 
+  appMetadata ? updateAppMetadata(appMetadata) : null
```

Comment:
```suggestion
  appMetadata && updateAppMetadata(appMetadata)
```

Reply:
This is a little more clean than a ternary that resolves to null. Resolving to null would make more sense if the ternary was used to set a variable.
A basic if block would be fine also